### ISSUE TITLE
Use of a custom model 

### ISSUE THREAD

**Issue by ChrisD-7** [davidshtian/Bedrock-ChatBot-with-LangChain-and-Streamlit#7]
- Pretty much new to using this wanted to fine tune and load my custom dataset.

- Login screen option for Google and others !!?

**Comment by ChrisD-7** [comment]
@davidshtian

**Comment by davidshtian** [comment]
> * Pretty much new to using this wanted to fine tune and load my custom dataset.
> * Login screen option for Google and others !!?

Hi~ This repo mostly focus on AWS Bedrock, you want to add some custom models? Do you have any example? Thanks~

**Comment by ChrisD-7** [comment]
Yes I have fine-tuned my own model and want to use that from hugging face as well @davidshtian

**Comment by ChrisD-7** [comment]
@davidshtian

**Comment by davidshtian** [comment]
> Yes I have fine-tuned my own model and want to use that from hugging face as well @davidshtian

Ideally, the fine-tuned HF model can be loaded and invoked, while this project is focusing on leveraging existing Bedrock models to further build something. For HF models, not sure if [chat-ui](https://github.com/huggingface/chat-ui) project could help. Thanks~
